new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -129,7 +131,7 @@
    */
   @BeforeAll
   public static void initAll(@Namespaces(4) List<String> namespaces) {
-
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.